app/views/static/buttons.html.haml

```diff
@@ -0,0 +1,45 @@
+.button-group
+  %button{:class     => "btn btn-primary btn-disabled",
```

Comment:
rubocop complains because this should probably be `%button.btn.btn-primary.btn-disabled` instead..